src/components/Favorites/index.tsx

```diff
@@ -5,10 +5,13 @@ import { RootState } from "../../redux/rootReducer";
 import "./Favorites.css";
 import FavoriteList from "./FavoriteList";
 
+const getFavoritesData = (state: RootState) => {
+  return state.favorites;
+};
+
 const Favorites = () => {
-  const favoritesData: FavoritesState = useSelector(
-    (state: RootState) => state.favorites
-  );
+  const favoritesData: FavoritesState = useSelector(getFavoritesData);
```

Comment:
Is `FavoritesState` type required? Isn't it inferred?

Reply:
`const favRepositories = favoritesData.favorites;`
For this line it is showing `Property 'favorites' does not exist on type 'never'. `

Reply:
I have resolved this by defining return type in `getFavoritesData ` function